src/metatrain/experimental/alchemical_model/default-hypers.yaml

```diff
@@ -26,5 +26,8 @@ architecture:
     log_interval: 5
     checkpoint_interval: 25
     per_structure_targets: []
-    loss_weights: {}
     log_mae: False
+    loss:
```

Comment:
I think we should also just allow

`loss: mse`

and exand this to

```
loss:
    type: mse
    weights: {}
    reduction: sum
```

This should help usability in is in line of what we do for the datasets.

Reply:
I think this is very hard to implement and maintain, plus it's confusing for the user, because then the hypers change dynamically. Since we give the user the full template of the hypers to rely on, I wouldn't do tricks

Reply:
I agree with @PicoCentauri here, having a smaller version of the same input is nice, especially if people are not just copy-pasting our templates around. We are already doing this for datasets:

```yaml
training_set: "dataset.xyz"
```

Actually means

```yaml
training_set:
    systems:
        read_from: dataset.xyz
        reader: ase
        length_unit: null
#        ...
```

Reply:
Copy-pasting the templates is the only way for users to modify the hypers... they can't make them up. This means that this feature, which is hard to implement, test and maintain, won't be used at all

Reply:
I'm saying that the overhead for this feature just isn't worth it, considering that the projected gain is from
```
loss:
  type: mae
```
to 
```
loss: mae
```

Reply:
For now, I think it opens too many possibilities and, from what I can see, it's not easy to test and maintain. Like in programming languages, having many ways of achieving the same thing is not always an advantage. I will open the issue though

Reply:
I don't see why the possibilities opened are too many, could you elaborate? Test & maintenance should also be trivial: if the type of `loss` is a string, you transform to `loss: {"type": <value>}` and then continue as is. Am I missing something here?

Reply:
I could understand if we where trying to be very explicit everywhere, but In general, metatrain `options.yaml` already includes a lot of syntactic sugar, from default hypers and with the datatset handling; and I really don't see how the loss would be any different.